EventFilter/HcalRawToDigi/plugins/PackerHelp.h

```diff
@@ -637,7 +637,7 @@ QIE11DataFrame convertHB(QIE11DataFrame qiehe, int tdc1, int tdc2) {
       tdc = 0;
     else if (tdc > tdc1 && tdc <= tdc2)
       tdc = 1;
-    else if (tdc2 < 49 && tdc > tdc2 && tdc <= 49)
+    else if (tdc2 < tdcmax && tdc > tdc2 && tdc <= tdcmax)
```

Comment:
```tdc2 < tdcmax``` is redundant here
(Given the sanity check you introduced before this method is called the decision structure could even be further simplified. However, I have nothing in contrary if the ```convertHB``` method can remain agnostic of the input parameters)

Reply:
Agreed and changed, since `convertHB` is unlikely to be used in other circumstances.